internal/generator/vector/output/loki/loki_test.go

```diff
@@ -3,26 +3,25 @@ package loki
 import (
 	"fmt"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/ginkgo/extensions/table"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
same here.   leave  dot reference so not needed to add `gingko.` to the below methods.